### ISSUE TITLE
[Question]

### ISSUE THREAD

**Issue by gschneller** [TheHive-Project/TheHive#2276]
### Request Type

Question

### Work Environment

| Question                   | Answer                                |
| -------------------------- | ------------------------------------- |
| OS version (server)        | Docker  thehiveproject/thehive4:4.0.5-1 |
| OS version (client)        | Ubuntu,           |
| Virtualized  Env.          | True /                          |
| Dedicated RAM              | 8 GB                                 |
| vCPU                       | 4                      |
| TheHive version / git hash | 4.0.5-1              |
| Package Type               | Docker|
| Database                   | Cassandra                |
| Index type                 | Lucene                 |
| Attachments storage        | Local                |
| Browser type & version     | Chrome: Version 96.0.4664.45                       |

### Question
Why aulogin don't work?

I configure Authentification with keycloak
```
user.autoCreateOnSso: true
user.profileFieldName: profile
user.organisationFieldName: organisation
user.defaults.organisation:example

auth {
  sso {
    autologin: true
    autocreate: true
  }
  providers: [
    {name: session}
    {name: basic, realm: thehive}
    {name: local}
    {name: key}
    {
      ##############
      # Keycloak
      ##############
      name: oauth2
      clientId: "thehive"
      clientSecret: "XXXXXXXXXXX"
      redirectUri: "http://example.com/thehive/api/ssoLogin"
      responseType: "code"
      grantType: "authorization_code"
      authorizationUrl: "http://example.com/keycloak/auth/realms/Reveelium/protocol/openid-connect/auth"
      authorizationHeader: "Bearer"
      tokenUrl: "http://keycloak:8080/auth/realms/Reveelium/protocol/openid-connect/token"
      userUrl: "http://keycloak:8080/auth/realms/Reveelium/protocol/openid-connect/userinfo"
      scope: ["openid", "email"]
      userIdField: "email"
    }
  ]
}
```
And at the fisrt conection, the TheHive GUI login is dispayed with the "Sign In with the SSO", I want tb be rediert directly on Keycloak login!

**Comment by X0x1RG9f** [comment]
Hi @gschneller, same problem here. Did you find a solution ? Looks like this parameter was removed from TH4, don't know why.

**Comment by mauli9999** [comment]
Yes @gschneller , In my case also TheHive GUI login is dispayed with the "Sign In with the SSO", I want tb be rediert directly on Keycloak login!
Does anyone have any solution on this?
